Billing/src/V1/CloudCatalogGrpcClient.php

```diff
@@ -2,7 +2,7 @@
 // GENERATED CODE -- DO NOT EDIT!
 
 // Original file comments:
-// Copyright 2020 Google LLC.
+// Copyright 2019 Google LLC.
```

Comment:
All the other files in this component have copyright 2020. I'm not sure this is what we want...